src/xml-parsers.js

```diff
@@ -201,12 +216,13 @@ const formatObjInfo = (content, opts = {}) => {
   const name = sanitizeObjectKey(toArray(Key)[0])
   const lastModified = new Date(toArray(LastModified)[0])
   const etag = sanitizeETag(toArray(ETag)[0])
+  const size = sanitizeSize(Size)
 
   return {
     name,
     lastModified,
     etag,
-    size: Size,
+    size,
```

Comment:
here only size is updated. but the PR title is for versionId. am i missing something ?
Could you clarify @fflorent  ?

Reply:
This PR disable number parsing of xml parser, so the real number (size) need to be parsed manually.